src/kotlin/org/intellij/plugins/hcl/terraform/config/model/ModuleDetectionUtil.kt

```diff
@@ -198,11 +199,6 @@ object ModuleDetectionUtil {
 
 
   private fun getTerraformModulesManifestFile(project: Project, dotTerraform: VirtualFile): VirtualFile? {
-    val projectRoot = project.baseDir
-    if (projectRoot != null && !VfsUtilCore.isAncestor(projectRoot, dotTerraform, false)) {
```

Comment:
Removed this check as it has already been done in `DotTerraformUtil.findTerraformDir()`.